test/e2e/framework/volume/fixtures.go

```diff
@@ -472,7 +477,17 @@ func testVolumeContent(f *framework.Framework, pod *v1.Pod, fsGroup *int64, fsTy
 // Multiple Tests can be specified to mount multiple volumes to a single
 // pod.
 func TestVolumeClient(f *framework.Framework, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
-	clientPod, err := runVolumeTesterPod(f.ClientSet, config, "client", false, fsGroup, tests)
+	testVolumeClient(f, config, fsGroup, fsType, tests, false)
+}
+
+// TestVolumeClientSlow is the same as TestVolumeClient, only it waits
+// for slowPodStartTimeout for the pod to start.
+func TestVolumeClientSlow(f *framework.Framework, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
+	testVolumeClient(f, config, fsGroup, fsType, tests, true)
+}
+
+func testVolumeClient(f *framework.Framework, config TestConfig, fsGroup *int64, fsType string, tests []Test, slow bool) {
+	clientPod, err := runVolumeTesterPod(f.ClientSet, config, "client", false, fsGroup, tests, slow)
```

Comment:
Could you add some comments about when to set slow to true or false?

Reply:
Added extensive docs to `TestVolumeClient` / `TestVolumeClientSlow`. I don't think it's necessary to document internal `testVolumeClient`.